tests/system_tests/plans.json

```diff
@@ -90,11 +96,13 @@
                         "description": "Mask contains points of axis within an xy circle of given radius.\n\n.. example_spec::\n\n    from scanspec.regions import Circle\n    from scanspec.specs import Line\n\n    grid = Line(\"y\", 1, 3, 10) * ~Line(\"x\", 0, 2, 10)\n    spec = grid & Circle(\"x\", \"y\", 1, 2, 0.9)",
                         "properties": {
                             "x_axis": {
+                                "axis": {},
```

Comment:
all the ScanSpec movables will change when the function arg in dodal is updated to be more accurate